src/charts/charts.less

```diff
@@ -67,7 +67,7 @@ pf-c3-chart {
   .col-sm-2 {
     padding-right: 0;
   }
-  .col-sm-10 {
+  .col-sm-10:not(.col-sm-pull-2) {
     padding-left: 0;
   }
```

Comment:
I would have expected this check to be on both columns and to do the reverse padding on them. That is, whichever column ends up on the left has a right-padding 0, and whichever ends up on the right has a left-padding of 0.

Reply:
@jeff-phillips-18 If I change the styling to this then the details is too close to the chart for the right label position as shown below unless I remove the ```  .col-sm-2.col-sm-push-10 { padding-left: 0; }```. Should I leave that specific style out?

```
.trend-card-compact-pf {
  .col-sm-2:not(.col-sm-push-10) {
    padding-right: 0;
  }
  .col-sm-2.col-sm-push-10 {
    padding-left: 0;
  }
  .col-sm-10:not(.col-sm-pull-2) {
    padding-left: 0;
  }
  .col-sm-10.col-sm-pull-2 {
    padding-right: 0;
  }
}
```

<img width="919" alt="screen shot 2017-08-25 at 6 04 30 pm" src="https://user-images.githubusercontent.com/20052391/29734459-53d84ea2-89c0-11e7-89b2-eafc17b7e040.png">

Reply:
@jeff-phillips-18 what are your thoughts?

Reply:
I guess I was assuming the text would be right justified in this layout. If not, then I guess we should leave the spacing as it was.

Reply:
@jeff-phillips-18 this is with ```text-align: right;``` on the ```trend-compact-details```. It doesn't seem to keep things lined up as far the 76TB and Storage are concerned, but it does fix the spacing from what I had in the above example. Not sure if we should leave it as I have it currently in this PR or use the styling above with the ```text-align: right;``` to fix the spacing.

<img width="237" alt="screen shot 2017-08-28 at 8 42 30 am" src="https://user-images.githubusercontent.com/20052391/29773787-f2135f0e-8bcc-11e7-8ac5-ff15516c0231.png">